teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super
```

Comment:
- [ ] I think here you missed some parameters so here when inheriting a class so also it inherit the paramters so please add the parameters for age, name and parent_permission with their defult values 💯🔥💪🏻😊

- [ ] As I can see also here you missed to pass the super() parameters because you inheirt from a class so also you should pass the parameters so please add it and pass base class parameters like above in student class 💪🏻🔥

it could be somthign like this 👍 
```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```